### DISCUSSION TITLE
Front-end editor: Which fields do you use?

### DISCUSSION THREAD

**Discussion by oliverklee** [oliverklee/ext-seminars#1378] [Ideas]
As part of the [compatibility with TYPO3 V11](https://github.com/oliverklee/ext-seminars/wiki/11LTS-Crowdfunding-Campaign), I will completely rewrite the frontend event editor. For this, I plan to only implement the inputs that are actually still used/needed. For this, I need your input. :-)

If you are using the FE event editor, please comment:
- Which of the existing inputs in the event editor form do you use (or plan to use)?

**Comment by lisardo2** [comment]
We dont use the frontend editor.

**Comment by oliverklee** [comment]
Two other customers of mine also do not use the FE editor.
